CHANGELOG.md

```diff
@@ -1,382 +1,341 @@
 # Changelog
-### [2.2.1](https://github.com/inovex/elements/compare/v2.2.0...v2.2.1) (2020-12-10)
 
+### [2.2.1](https://github.com/inovex/elements/compare/v2.2.0...v2.2.1) (2020-12-10)
 
 ### Bug Fixes
 
-* **elements|ino-datepicker:** use date format to improve validation ([#226](https://github.com/inovex/elements/issues/226)) ([90a990c](https://github.com/inovex/elements/commit/90a990c32c92c0ebd1ad88613aafa99513d54d70))
-
+- **elements|ino-datepicker:** use date format to improve validation ([#226](https://github.com/inovex/elements/issues/226)) ([90a990c](https://github.com/inovex/elements/commit/90a990c32c92c0ebd1ad88613aafa99513d54d70))
```

Comment:
This might be a problem because the changelogs are generated automatically. Can we exclude them?

Reply:
Good point! I added the changelogs to the prettierignore config and restored the previous changelogs.